### PR TITLE
Copy CRDs one at a time

### PR DESCRIPTION
**Pull request by TerryHowe**
Copy CRD into bucket

### CODE REVIEW

File: release/s3_sync.sh [hunk 1]
```diff
@@ -28,9 +28,19 @@ else
   PUBLIC_READ=''
 fi
 cd ${BASE_DIRECTORY}
-DEST_DIR=kubernetes-${RELEASE_BRANCH}/releases/${RELEASE}/artifacts
-if [ ! -z "${REPO}" ]
+BASE_DIR=kubernetes-${RELEASE_BRANCH}
```
Comment (abhay-krishna) [inline release/s3_sync.sh:31]: Can we name this something along the lines of PREFIX_DIR or RELEASE_BASE_DIR?

File: Makefile [hunk 2]
```diff
@@ -73,7 +73,7 @@ postsubmit-conformance:
 .PHONY: upload
 upload:
 	release/generate_crd.sh $(RELEASE_BRANCH) $(RELEASE)
-	release/s3_sync.sh $(RELEASE_BRANCH) $(RELEASE) $(ARTIFACT_BUCKET)
+	release/s3_sync.sh $(RELEASE_BRANCH) $(RELEASE) $(ARTIFACT_BUCKET) ""
```
Comment (abhay-krishna) [inline Makefile:76]: I think it works even if you don't pass in the quotes here. As far as I have seen, Bash automatically takes missing arguments to be empty strings.

File: release/s3_sync.sh (hunk 1 above)
Comment (TerryHowe) [reply release/s3_sync.sh]: Done

**Review by abhay-krishna — APPROVED** [review]
/lgtm
/approve

**Pull request merged**